pyelastix.py

```diff
@@ -559,21 +588,23 @@ def _write_image_data(im, id):
     The id is the image sequence number (1 or 2). Returns the path of
     the mhd file.
     """
-    im = im* (1.0/3000)
+    # im = im * (1.0/3000)  # TODO: WTF is this?
```

Comment:
@almarklein could you comment on this line?

Reply:
wow ... dunno. My first guess is that it converts CT data to `0..1` range, kinda. (The division inside parenthesis is done because multiplication of each element is faster than division.)

Does commenting the line has any effect on registration results? I think it *does* always change the dtype of `im` to float.

Reply:
I was playing with registration of float images (as in the added test) and this line causes the range of the resulting image to be 3000 smaller for no apparent reason. Additionally, in my specific case, this cause the intensity levels to go below 0.

People reported this issue in https://github.com/almarklein/pyelastix/issues/6 and https://github.com/almarklein/pyelastix/issues/19.

I can add a test case for ubyte images if that helps. But, in any case, that line does not look right.

Reply:
> But, in any case, that line does not look right.

I completely agree :)